### ISSUE TITLE
Memtest86+ ECC memory support

### ISSUE THREAD

**Issue by UttarwarSunil** [memtest86plus/memtest86plus#246]
Hi Team,

I see that MemTest86 from Passmark support ECC memory and reports the errors as shown in below snapshot.
![ecc-memtest86](https://user-images.githubusercontent.com/70257439/215079254-f12974ff-d029-4d40-98f0-c3cc1836be64.png)

Do we have ECC memory support in Memtest86+? Can you please confirm MemTest86+ also reports similar ECC memory errors?

Thanks
Sunil

**Comment by debrouxl** [comment]
Nope, reporting ECC-corrected memory errors is not currently supported in memtest86+. See #92 , which you can upvote if EDAC support is really important to you :)
